config/base_boxes.yaml

```diff
@@ -3,27 +3,47 @@ boxes:
     box_name:   'debian/jessie64'
     image_name: !ruby/regexp '/Debian.*Jessie/'
     pty:        true
+    scenarios:
+      - foreman
 
   debian9:
     box_name:   'debian/stretch64'
     image_name: !ruby/regexp '/Debian.*Stretch/'
     pty:        true
+    scenarios:
+      - foreman
 
   centos6:
     box_name:   'centos/6'
     image_name: !ruby/regexp '/CentOS 6.*PV/'
     pty:        true
+    scenarios:
+      - foreman
+      - katello
```

Comment:
Do we want the base boxes to have scenarios? For example, what if I just want a centos6 box?